src/stream.js

```diff
@@ -0,0 +1,264 @@
+import { encodeEntities, styleObjToCss, assign, getChildren } from './util';
+import { options, Fragment } from 'preact';
+import stream from 'stream';
+
+// components without names, kept as a hash for later comparison to return consistent UnnamedComponentXX names.
+const UNNAMED = [];
```

Comment:
Note to self: Seems like a memory leak when keeping references to components around in a global variable